patterns/lcesave.hexpat

```diff
@@ -1,17 +1,16 @@
 #pragma author DexrnZacAttack
-#pragma description Decompressed Minecraft LCE Save File
+#pragma description Minecraft LCE Save File
 
-#pragma endian big
-/* ^ switch this to little for Switch, PS4, or Xbox One */
 
-
-// NOTE: This pattern was only tested on Nightly@974c4ba, things may break/not work on older versions.
-// ALSO NOTE: SPEGHETTI CODE!
-
-// TODO: re-add nbt pattern
-// TODO: clean up getFileType
+// NOTE: SPEGHETTI CODE!
 
 import std.string;
+import std.mem;
+import std.core;
+import hex.dec;
+#ifdef __IMHEX__
```

Comment:
Where does the `__IMHEX__` define come from?

Reply:
It is defined internally before pattern is evaluated. All patterns have it.

Reply:
DexrnZacAttack, why are you testing for this define?

Reply:
It isn't my place to reply since you asked the author directly, but note that the commit that adds the test for the define has "ifdefs to fix actions moment " as comment, so I suspect that it has to do with unit tests which wouldn't define the __IMHEX__ variable because tests don't use the runtime. Also it would make little sense to warn user during a test run which is supposed to run automatically.

Reply:
Thanks Paxcut.

Reply:
> Thanks, Paxcut.
Just saw this thread, yes this is the case, as the actions stuff was failing without that check.